install-2x.html

```diff
@@ -113,24 +113,24 @@ <h4>2. Download and prepare the code</h4>
 cd cypht-temp
 
 # grab latest code
-wget https://github.com/cypht-org/cypht/archive/master.zip
+wget https://github.com/cypht-org/cypht/archive/refs/tags/v2.0.0.zip
```

Comment:
So we will need to update this for each release? Surely there is a way to get URL of latest 2.x as per https://github.com/cypht-org/cypht/wiki/Lifecycle

Reply:
Thanks @marclaporte, Coming back with an update.